arguments.py

```diff
@@ -0,0 +1,29 @@
+import sys
+import argparse
+
+
+def parameters():
+    parser = argparse.ArgumentParser(
+            prog='BetterWeather',
+            description="""This program is written in Python.
+            Its purpose is to collect weather forecast data
+            from a few free weather forecast APIs (e.g. Open-Meteo, WeatherAPI)
+            and combine it into one more accurate weather prediction.
+            After providing the location eg. "Madrid" you will receive a file
+            with weather data and a user-friendly plot.""")
```

Comment:
That's not fully true yet, isn't it?

Reply:
Yes, but soon enough so I am not changing it :)

Reply:
Ok, let it be.